applications/TuxedoDapp

```diff
@@ -0,0 +1,232 @@
+
+# TuxedoDapp
+
+- **Team Name:** MLabs
+- **Payment Address:** FIAT
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview :page_facing_up:
+
+Develop a Tuxedo Web Wallet and Dapp
+
+### Overview
+
+Unlike traditional account-based systems, UTXO-based systems treat each transaction output as a distinct entity, introducing advantages in parallel processing and state transition simplicity. 
+The Tuxedo project endeavours to demonstrate the flexibility and interoperability of Polkadot by implementing UTXO as one of its paradigms. MLabs, with expertise in the Cardano ecosystem utilizing UTXO as its core model, sees an opportunity to contribute to Tuxedo's success, address identified issues, and engage in future collaborative activities.
+
+Our goal is to showcase the potential of Polkadot using the UTXO paradigm for the end user, leveraging Tuxedo. The project will consist of two key stages:
+
+- Stage 1: Developing a simple Web Wallet
+- Stage 2: Developing a basic Dapp
+
+### Project Details
+
+- Stage 1: Developing a simple Web Wallet
+
+Develop a user-friendly web wallet that allows users to manage their UTXOs on the Tuxedo-based Polkadot network which is similar to polkadot.js which will get the chain info(genesisHash, runtime metadata, chain runtime version, API version), block info
+Focus on simplicity, security, and ease of use to provide a seamless experience for users interacting with UTXOs.
+
+**Technical Requirements** 
+ 
+**Security:** : Private key storage:
+Web-based wallet will provide a secure method for private key storage on the client side, such as a browser's secure storage or hardware wallets.
+
+**Functionality**: 
+1. Feature Parity Between Web-Based and CLI Wallets
+The web-based wallet will be designed to mirror the core functionality of the CLI wallet, offering seamless UTXO synchronization and token handling, ensuring that users have an equivalent experience across both interfaces 
+
+2. Issues Fix in web-based wallet :(issues/62)
+The current implementation of a cli based wallet allows sending inputs from multiple owners in a single transaction, but it only supports a single recipient for all specified outputs.
+
+3. Browser compatibility:
+Web-based wallet will ensure compatibility with at least one browser (Chrome, Firefox or Edge).
```

Comment:
Ideally you'd support at least the most popular browser, see https://gs.statcounter.com/browser-market-share/

Reply:
Yes we will support Chrome, Firefox and Edge.
We will update the below table .